web/src/components/molecules/Common/Header/index.tsx

```diff
@@ -43,7 +45,7 @@ const Header: React.FC<Props> = ({
     <HeaderMenu
       items={[
         {
-          label: "Personal Account",
+          label: <>{t("Personal Account")}</>,
```

Comment:
Pretty sure either `t("Personal Account")` or `{t("Personal Account")}` will work. Don't need the fragment